docs/dev/docker.rst

```diff
@@ -0,0 +1,45 @@
+Docker Setup for local development
+==================================
+The following describes how to set up an instance of the site on your
+computer for development using the provided docker and docker-compose 
```

Comment:
This says "for development", but the docs below don't explain below how to use the started docker-compose environment to develop the code. Can you add some details about how to make code changes, set up the database, configure, and configure the environment?

Alternatively, if the above things aren't what you meant this to be used for, this paragraph could change to better reflect the goals of this docker-compose set up.

Reply:
Well we could have them build each time, or have a volume mount to the docker image that is built. I think the volume would be better as building could take a while and use a lot of resources.

Reply:
Looking at this again, this is the reason I wanted to use docker-compose and I would suggest others do too. I don't need to have the user do any additional setup. They can literally start the compose file and get to work with their changes. I don't feel any other documentation about the database or environments is needed at this point as the docker-compose.yml file does all the setup

Reply:
I expected more set up would be needed to get a working site, like creating a user account. I see now that `first-start` handles the bulk of that. I think these docs leave a pretty big "what now?" question about where to go if you aren't already familiar with docker-compose or Normandy, but that might be ok.

Reply:
I could point to steps 6-11 here: https://mozilla.github.io/normandy/dev/install.html#installation and say they need to be done within docker using a command